### ISSUE TITLE
Create a new stack

### ISSUE THREAD

**Issue by FJMentz** [TripleParity/docks-ui#14]
Functionality (linked to a button) to add a new stack? If I understand correctly this will basically just be adding multiple containers linked to one stack? 

@devosray can you verify or explain if I went off track?

**Comment by devosray** [comment]
Think of it as instead of uploading a Docker file or image tag to be created, you upload a whole Docker-compose file. Here is a really good example: https://docs.docker.com/get-started/part5/#add-a-new-service-and-redeploy

**Comment by FJMentz** [comment]
Front-end is paused and just waiting on backend.

**Comment by devosray** [comment]
Working on it right now.

**Comment by egeldenhuys** [comment]
Duplicate of #114 . Closing this as the new issue has more detail.
